frontend/pages/faq.tsx

```diff
@@ -14,53 +15,37 @@ import HeaderImage from "../src/components/staticpages/HeaderImage";
 
 const useStyles = makeStyles((theme) => {
   return {
-    headerImageContainer: {
-      marginBottom: theme.spacing(3),
-      [theme.breakpoints.down("lg")]: {
-        marginBottom: theme.spacing(2),
-      },
-    },
     headerTextContainer: {
       position: "absolute",
-      top: "50%",
-      left: "50%",
-      transform: "translate(-50%, -50%)",
-      background: "rgba(255, 255, 255, 0.6)",
-      width: "100%",
-      height: "100%",
-      [theme.breakpoints.down("lg")]: {
-        background: "rgba(255, 255, 255, 0.8)",
-      },
+      top: "45%",
+      left: "10%",
+      maxWidth: "400px",
+      minWidth: "250px",
+      borderRadius: "5px",
+      backgroundColor: "#207178",
     },
     headerTextInnerContainer: {
       height: "100%",
       display: "flex",
```

Comment:
```suggestion
      borderRadius: "5px",
      backgroundColor: "#207178",
      paddingRight: "25px",
```
See above. This is now the "new card" while the one above is the line wrapper.